elixir/apps/web/lib/web/live/policies/show.ex

```diff
@@ -144,11 +144,20 @@ defmodule Web.Policies.Show do
         Danger zone
       </:title>
       <:actions>
-        <.delete_button>
+        <.delete_button phx-click="delete" phx-value-id={@policy.id}>
           Delete Policy
         </.delete_button>
       </:actions>
     </.header>
     """
   end
+
+  def handle_event("delete", %{"id" => id}, socket) do
+    with {:ok, policy} <- Policies.fetch_policy_by_id(id, socket.assigns.subject) do
+      {:ok, _} = Policies.delete_policy(policy, socket.assigns.subject)
+      {:noreply, redirect(socket, to: ~p"/#{socket.assigns.account}/policies")}
```

Comment:
Same question here Re: `redirect/2` vs `push_navigate/2`.

Reply:
Same response as above.  Will get this updated.

Reply:
Updated this to use `push_navigate`